requirements.txt

```
@@ -1,2 +1,4 @@
 beautifulsoup4==4.11.1
+lxml==4.9.1
+requests==2.28.1
```

```suggestion
requests==2.27.1
```
2.28.1 doesnt support 3.6

---

Updated in [1e819dd](https://github.com/cjquines/firehose/pull/9/commits/1e819ddace36f4f5aaa043e52944fbec1d3d55a2)